src/Models/Environment.php

```diff
@@ -951,7 +949,7 @@ protected function parseAttributes($data)
     private function settings($setting = null)
     {
         $path = "sites/{$this->getSite()->id}/environments/{$this->id}/settings";
-        $response = (array)$this->request()->request($path, ['method' => 'get',]);
```

Comment:
is there a reason to remove this array cast?

Reply:
I think this may be a long-standing bug with this function with newer versions of PHP. Casting to an array caused the response to not have any data in `$response['data']` because the resulting cast'd array had something like `$response['Pantheon\Terminus\Request\RequestOperationResult data']` instead